### PR TITLE
feat(v2): initial development

### PR DESCRIPTION
**Pull request by jgolden17**
This version allows for finer control over SEO metadata and adds support of all Open Graph and Twitter Card types.

BREAKING CHANGE

### CODE REVIEW

**Comment by jgolden17** [comment]
@amexjake @igetgames Can I get reviews please

File: README.md [hunk 1]
```diff
@@ -25,26 +25,87 @@ Want to get paid for your contributions to `react-seo`?
...
+Let's start with a minimal example of basic usage:
+
 ```javascript
 import React from 'react';
 import SEO from '@americanexpress/react-seo';
 
 const MyModule = () => (
   <div>
     <SEO
-      author="John Doe"
```
Comment (infoxicator) [inline README.md:35]: @jgolden17 how do you pass the author now?

**Comment by jgolden17** [comment]
> @jgolden17 Any plans to also add microdata (schemas)?

That's a good idea. Can you create an issue for it?

File: README.md (hunk 1 above)
Comment (jgolden17) [reply README.md:35]: I haven't added a schema type of articles yet.  Will have a fast follower to add that.

**Pull request merged**